SecretManager/src/V1/Client/SecretManagerServiceClient.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * Generated by gapic-generator-php from the file
+ * https://github.com/googleapis/googleapis/blob/master/google/cloud/secretmanager/v1/service.proto
+ * Updates to the above are reflected here through a refresh process.
+ */
+
+namespace Google\Cloud\SecretManager\V1\Client;
+
+use Google\Cloud\SecretManager\V1\Client\BaseClient\SecretManagerServiceBaseClient;
+
+/**
+ * {@inheritdoc}
+ *
+ * This class is currently experimental and may be subject to changes.
```

Comment:
This should also have the `@experimental` tag

Reply:
So weird, it's [right here](https://github.com/googleapis/gapic-generator-php/blob/v2-dev/src/Generation/EmptyClientV2Generator.php#L59)...will need to look into it.

Reply:
Apparently the synth scripts removed it:
`synthtool [INFO] > Replaced '^(\\s+\\*\\n)?\\s+\\*\\s@experimental\\n' in src/V1/Client/SecretManagerServiceClient.php`

This is what I see after running the `owlbot-php` processor.

Reply:
Which makes sense because the `@experimental` tag appears in the generator baselines. The only difference is that _this_ code is post-processed by OwlBot.

Reply:
@bshaffer can you take a look at that? I'm not sure where that stuff is implemented/what's expected.

Reply:
Oh it's because there is a pretty loose replace statement in the `owlbot.py` for a few libs. I will see what I can do.

Reply:
That shouldn't be necessary because the v1 client generator will only add `@experimental` if the proto package includes `alpha` or `beta` in it, which this service does not. 

https://github.com/googleapis/gapic-generator-php/blob/main/src/Generation/GapicClientGenerator.php#L122
https://github.com/googleapis/gapic-generator-php/blob/main/src/Generation/ServiceDetails.php#L307

https://github.com/googleapis/google-cloud-php/blob/35b4831c11bd0649211f84ff5a21bc5c9e93c33f/SecretManager/owlbot.py#L34-L38
I think this can be safely removed from at HEAD. WDYT?

Reply:
I can confirm that removing it altogether doesn't affect the existing v1 GAPIC

Reply:
https://github.com/googleapis/google-cloud-php/pull/5976